pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -93,6 +102,10 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) (opErr error) {
 
 	ctx.Logger.V(2).Info("deleting cluster")
 
+	if err := a.deleteKubeConfig(ctx); err != nil {
+		return errors.Wrapf(err, "error deleting kubeconfig for cluster %q", ctx)
```

Comment:
Hi @andrewsykim,

Should this be a requeue error? I'm not sure, which is why I'm asking.